tests/Rules/Data/model-property-builder.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+\App\User::query()->firstWhere('foo', 'bar');
```

Comment:
I think it would be nice to test a  column with a table prefix as well

Reply:
If there is a `.` in the column name we currently skip it. So I didn't add test for it.